CONTRIBUTING.md

```diff
@@ -6,42 +6,63 @@ Thank you for your interest in contributing to Okta's Open Source Projects! Befo
 
 ### Git Commit Messages
 
-We use a simplified form of [Atom's](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages) commit convention.
+We use an adapted form of [Conventional Commits](http://conventionalcommits.org/).
 
   * Use the present tense ("Adds feature" not "Added feature")
```

Comment:
Opinions about `Adds`(present tense) vs `Add`(Infinitive)? I personally prefer `Add`(Infinitive), and I noticed that in this repo we usually use the infinitive form, but haven't changed this yet in this doc. What do you guys think?

Reply:
I personally like `Adds`, but will go with whatever the team prefers.

Reply:
I don't have an opinion on this.